src/explore-education-statistics-frontend/src/modules/find-statistics/__tests__/FindStatisticsPageNew.test.tsx

```diff
@@ -446,4 +453,402 @@ describe('FindStatisticsPageNew', () => {
+      })
+      .mockResolvedValueOnce({
+        results: testPublications,
+        paging: testPaging,
+      });
+
+    render(<FindStatisticsPageNew />);
+
+    await waitFor(() => {
+      expect(screen.getByText('1 result')).toBeInTheDocument();
+    });
+
+    expect(screen.getByText('Page 1 of 1, filtered by:')).toBeInTheDocument();
+    expect(
+      screen.getByRole('button', {
+        name: 'Remove filter: Find me',
+      }),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole('button', { name: 'Clear all filters' }),
+    ).toBeInTheDocument();
+    expect(
+      within(screen.getByTestId('publicationsList')).getAllByRole('listitem'),
+    ).toHaveLength(1);
+
+    userEvent.click(
+      screen.getByRole('button', {
+        name: 'Remove filter: Find me',
+      }),
+    );
+
+    expect(mockRouter).toMatchObject({
+      pathname: '/find-statistics',
+      query: {},
+    });
+
+    await waitFor(() => {
+      expect(screen.getByText('30 results')).toBeInTheDocument();
+    });
+
+    expect(
+      screen.getByText('Page 1 of 3, showing all publications'),
+    ).toBeInTheDocument();
+    expect(
+      screen.queryByRole('button', { name: 'Remove filter: Find me' }),
+    ).not.toBeInTheDocument();
+    expect(
+      screen.queryByRole('button', { name: 'Clear all filters' }),
+    ).not.toBeInTheDocument();
+    expect(
+      within(screen.getByTestId('publicationsList')).getAllByRole('listitem'),
+    ).toHaveLength(3);
+  });
+
+  test('sorting', async () => {
+    publicationService.listPublications.mockResolvedValue({
+      results: testPublications,
+      paging: testPaging,
+    });
+
+    render(<FindStatisticsPageNew />);
+
+    await waitFor(() => {
+      expect(screen.getByText('30 results')).toBeInTheDocument();
+    });
+
+    const sortGroup = within(
+      screen.getByRole('group', { name: 'Sort results' }),
+    );
+    const sortOptions = sortGroup.getAllByRole('radio');
+    expect(sortOptions).toHaveLength(3);
+    expect(sortOptions[0]).toEqual(sortGroup.getByLabelText('Newest'));
+    expect(sortOptions[0]).toBeChecked();
+    expect(sortOptions[1]).toEqual(sortGroup.getByLabelText('Oldest'));
+    expect(sortOptions[1]).not.toBeChecked();
+    expect(sortOptions[2]).toEqual(sortGroup.getByLabelText('A to Z'));
+    expect(sortOptions[2]).not.toBeChecked();
+
+    userEvent.click(sortGroup.getByLabelText('A to Z'));
+
+    await waitFor(() => {
+      expect(mockRouter).toMatchObject({
+        pathname: '/find-statistics',
+        query: { sortBy: 'title' },
+      });
+    });
+
+    expect(sortOptions[0]).toEqual(sortGroup.getByLabelText('Newest'));
+    expect(sortOptions[0]).not.toBeChecked();
+    expect(sortOptions[1]).toEqual(sortGroup.getByLabelText('Oldest'));
+    expect(sortOptions[1]).not.toBeChecked();
+    expect(sortOptions[2]).toEqual(sortGroup.getByLabelText('A to Z'));
+    expect(sortOptions[2]).toBeChecked();
+  });
+
```

Comment:
Would be good to get some tests in here for at least the following (there might be some others in this general area):

- sorting with search (i.e. relevance)
- removing relevance sorting when search is cleared